clickhouse_search/search_tests.py

```diff
@@ -403,17 +404,14 @@ def test_quality_filter(self):
         #     annotations=annotations, pathogenicity={'clinvar': ['pathogenic']},
         # )
 #
-#     def test_location_search(self):
-#         self._assert_expected_search(
-#             [MULTI_FAMILY_VARIANT, VARIANT4], omit_data_type='SV_WES', **LOCATION_SEARCH,
-#         )
-#
-#         # Test "large" gene list search
```

Comment:
removing this test as the clustering logic for large gene lists has been removed